src/main/java/com/facebook/openwifirrm/RRMAlgorithm.java

```diff
@@ -143,6 +143,8 @@ public static RRMAlgorithm parse(String name, String argsRaw) {
 	 * @param dryRun if set, do not apply changes
 	 * @param allowDefaultMode if false, "mode" argument must be present and
 	 *                         valid (returns error if invalid)
+	 * @param updateImmediately whether to interrupt the config manager thread
+	 * to trigger immediate update
```

Comment:
Also mention that the zone gets marked for update (that's important)
style: we usually indent `@param` overflow lines, see the param above (I don't really care, just be consistent)